src/commands/skullboard.py

```diff
@@ -263,6 +275,204 @@ async def hof(self, interaction: Interaction):
+
+            # Collecting stats
+            count = sum([y for x, y in data])  # number of posts in total
+            above_threshold = sum(
+                [y for x, y in data if x >= self.db.threshold]
+            )  # number of posts meeting or exceeding threshold
+            percentile = round(
+                100 * above_threshold / count, 1
+            )  # percentile of posts meeting or exceeding threshold
+
+            msg = [
+                f"### {title}:",
+                f"Count: **{count}**",
+                f"Over threshold: {above_threshold}/{count} = **{percentile}%**",
+            ]
+            msg = "\n".join(msg)
+
+            embed = Embed(
+                title="💀💀💀   SKULL STATS   💀💀💀",
+                colour=LIGHT_GREY,
+                description=msg,
+            )
+
+            # generate a histogram of skull reaction counts for posts in the time period
+            curr_time = time.get_timestamp_str()
+            img_name = (title + " " + curr_time + ".png").replace(" ", "_")
+            img_buf = get_histogram_image(data)
+            img = File(img_buf, filename=img_name, description=title)
+            embed.set_image(url="attachment://" + img_name)
+
+            await interaction.response.send_message(
+                file=img,
+                embed=embed,
+                allowed_mentions=AllowedMentions().none(),
+                ephemeral=True,
+            )
+
+        except Exception as e:
+            logging.exception("Stats")
+            await interaction.response.send_message(
+                f"An error occurred: {str(e)}", ephemeral=True
+            )
+
+    @app_commands.command(name="user", description="Get user stats")
+    async def user(self, interaction: Interaction, user: Member):
+        try:
+            user_id = user.id
+            user_name = user.name
+
+            data = await self.db.get_user_rankings(999999)  # get all
+            data = [(id, freq) for id, freq in data if freq > 0]
+
+            if not data:
+                await interaction.response.send_message(
+                    "Database error or no posts saved - check the logs.",
+                    ephemeral=True,
+                )
+                return
+
+            user = [(id, freq) for id, freq in data if id == user_id]
+            if not user:
+                user = [(user_id, 0)]
+            _, user_skull_count = user[0]
+
+            # skull stats
+            count = len(data)  # count the number of users
+            values = [
+                freq for _, freq in data
+            ]  # number of skullboarded posts for each user
+            above_count = len(
+                [1 for val in values if val >= user_skull_count]
+            )  # how many users have as much or more skull reaction posts than the user
+            percentile = round(
+                100 * above_count / count, 1
+            )  # top percentile of skull reactions
```

Comment:
Put the comments in the line above the statement to prevent wrapping like this when formatted by Black